octoprint/datadog_checks/octoprint/check.py

```diff
@@ -0,0 +1,138 @@
+# After installing the Datadog Agent on your OctoPrint device, create an API
+# key from the OctoPrint service, and provide it.
+# Drop this file into `/etc/datadog-agent/checks.d` along with an empty file
+# named `octoprint.yaml` in `/etc/datadog-agent/conf.d`.  Pass in your
+# OctoPrint API key as an instance value like so:
+# ```
+# instances:
+#   - octo_api_key: APIKEYGOESHERE
+# ```
+
+# TODO:
+# - Refactor Octo HTTP API calls
+
+import logging
+import os
+
+import requests
+
+from datadog_checks.base import AgentCheck
+from datadog_checks.base.utils.subprocess_output import get_subprocess_output
+
+__version__ = "0.1.2"
+
```

Comment:
```suggestion
```

Versions get handled by the `__about__.py` file.

Reply:
Thank you!  I wasn't sure about that.  Probably cargo-culted that.